CMakeLists.txt

```diff
@@ -435,114 +480,84 @@ function(is_sdk_requested name result_var_name)
   endif()
 endfunction()
 
-# FIXME: separate the notions of SDKs used for compiler tools and target
-# binaries.
-if("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
-  set(CMAKE_EXECUTABLE_FORMAT "ELF")
+# Required from this point:
+# - SWIFT_HOST_VARIANT_SDK  - OS of system which will host the compiler
+# - SWIFT_HOST_VARIANT_ARCH - arch of system which will host the compiler
```

Comment:
Sorry, I don't understand the point of the first part of this comment.  Doesn't it repeat what was already explained in the code above?  (In the `set(...CACHE...)` part.)